lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -21,10 +21,21 @@ class BaseI18nTestCase(TestCase):
     """
     Base utilities for i18n test classes to derive from
     """
+    def __init__(self, *args, **kwargs):
```

Comment:
Yes, sometimes quality complains about initializing variables outside of the init method.  However, it does special case the `setUp` method and allow for variable initialization within setUp.  If your test really needs to initialize outside of `setUp` then you can quiet pylint.